### ISSUE TITLE
find-my-way update, 404 handler issue

### ISSUE THREAD

**Issue by AVVS** [fastify/fastify#324]
Original description - https://github.com/delvedor/find-my-way/issues/32

I assume that either find-my-way should allow `/*` routes, or it should simply be a `*` route and for cases where we want to add prefix (do we?), should just do `(prefix ? prefix + '/' + star : star)`

**Comment by delvedor** [comment]
Hi! Sorry for the delay.
I didn't understand the issue, can you rephrase?

Thanks!

**Comment by delvedor** [comment]
Closing for inactivity :)
